components/lwip/port/esp32/include/lwipopts.h

```diff
@@ -1074,6 +1074,15 @@ LWIP_FORWARD_DECLARE_C_CXX void sntp_sync_time(struct timeval *tv);
 
 LWIP_FORWARD_DECLARE_C_CXX uint32_t sntp_get_sync_interval(void);
 
+// Max number of SNTP servers handled (default equal to LWIP_DHCP_MAX_NTP_SERVERS)
+#if defined CONFIG_LWIP_SNTP_MAX_SERVERS && !defined SNTP_MAX_SERVERS
```

Comment:
```suggestion
#if defined CONFIG_LWIP_SNTP_MAX_SERVERS
```
As we don't allow to override `CONFIG_...` macros defined in configuration menu.

Reply:
OK, I will amend the change into last commit to keep it clean.